pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -87,6 +99,18 @@ func (pa *PodAutoscaler) Target() (target int32, ok bool) {
 	return 0, false
 }
 
+// RawTarget returns the raw target annotation value or false if not present. It's used in custom metric assignment.
+func (pa *PodAutoscaler) RawTarget() (target string, ok bool) {
```

Comment:
you don't really have to name them here, I think and just make the body `return pa.Annotations...`. 
The same below.

Reply:
The compiler errors when returning a map value from a function with a return signature of `(T, bool)`.  Is there a trick here for which I'm unfamiliar?

```
pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go|109 col 2| wrong number of return values (want 2, got 1) (typecheck)
||      return pa.Annotations[autoscaling.MetricNameAnnotationKey]
||      ^
```

Reply:
I guess the compiler is not smart enough to infer which getter to call from the context 🤦‍♂ 
Sorry for confusion :|